### PR TITLE
gh-107091: Fix some uses of :attr: role

### PR DESCRIPTION
**Pull request by serhiy-storchaka**
`:c:member:` should be used instead.

* Issue: gh-107091

----
:books: Documentation preview :books:: https://cpython-previews--107318.org.readthedocs.build/

### CODE REVIEW

File: Doc/c-api/import.rst [hunk 1]
```diff
@@ -310,23 +310,25 @@ Importing Modules
...
-   The structure is defined in :file:`Include/import.h` as::
+   The structure consists of two members:
 
-      struct _inittab {
-          const char *name;           /* ASCII encoded string */
-          PyObject* (*initfunc)(void);
-      };
+   .. c:member:: const char *name
+
+      the module name, as an ASCII encoded string
```
Comment (vstinner) [inline Doc/c-api/import.rst]: Please put an uppercase at the start of the sentence and a dot at the end, same below:

```suggestion
      The module name, as an ASCII encoded string.
```

File: Doc/c-api/init.rst [hunk 2]
```diff
@@ -827,8 +827,11 @@ code, or when embedding the Python interpreter:
 .. c:type:: PyThreadState
 
    This data structure represents the state of a single thread.  The only public
-   data member is :attr:`interp` (:c:expr:`PyInterpreterState *`), which points to
-   this thread's interpreter state.
+   data member is:
+
+   .. c:member:: PyInterpreterState *interp
+
+      this thread's interpreter state
```
Comment (vstinner) [inline Doc/c-api/init.rst]:
```suggestion
      This thread's interpreter state.
```

File: Doc/c-api/import.rst [hunk 3]
```diff
@@ -310,23 +310,25 @@ Importing Modules
...
+
+   .. c: member:: PyObject* (*initfunc)(void)
+
+      initialization function for a module built into the interpreter
```
Comment (vstinner) [inline Doc/c-api/import.rst]:
```suggestion
      Initialization function for a module built into the interpreter.
```

**Review by vstinner — APPROVED** [review]
LGTM. I just have a minor suggestion.

File: Doc/c-api/import.rst (hunk 1 above)
Comment (serhiy-storchaka) [reply Doc/c-api/import.rst]: Thank you. I initially wrote this in such way, but then I seen that lowercase without a period is used for `PyMethodDef` members (8edcb30c3f8bdd8099a093146fedbd9b63a3f667), and I change it back. Now I see that descriptions of all other members are written as full sentences, so I think that `PyMethodDef` members should be fixed.

**Pull request merged**